README.md

```diff
@@ -2,17 +2,23 @@
 
 Useful reusable components to use in the [Maji][maji] framework.
 
-## Installation:
+## Installation
 
 `npm i maji-extras --save`
 
-## Usage:
+## Usage
 
 You'll never use all of maji-extras, but you pick exactly the parts that you need for your project.
 
+## Development
+
+* Run `npm install` to install all dependencies.
+* Run `bin/karma start` to run all JavaScript tests.
```

Comment:
see: https://github.com/kabisa/maji-extras/blob/master/package.json#L8

`npm run watch` will trigger a karma start
`npm test` will run all tests and linters